routes/middlewares/festival/lol/get/getLol.ts

```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from 'express';
+
+import CustomError from '@Middleware/error/customError';
+import LolTeam from '@Model/lolTeam.model';
+import LolTeamMember from '@Model/lolTeamMember.model';
+import LolVote from '@Model/lolVote.model';
+
+const getLol = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const result: LolTeam[] = await LolTeam.findAll({
+      include: [
+        {
+          model: LolTeamMember,
+          attributes: ['pk', 'team_pk', 'name', 'studentId', 'leader'],
+        },
+        {
+          model: LolVote,
+          attributes: ['pk', 'team_pk', 'user_pk', 'createdAt'],
```

Comment:
```suggestion
          attributes: ['team_pk'],
```
user_pk를 넣었다는건 엄청 큰 실수고 서버에서 투표율 % 계산해서 응답해줘야됨